gnomad_qc/v4/resources/release.py

```diff
@@ -196,3 +199,24 @@ def release_coverage(
             for release in COVERAGE_RELEASES[data_type]
         },
     )
+
+
+def hgdp_tgp_updated_callstats(
+    test: bool = False, subset: str = "final"
+) -> VersionedTableResource:
+    """
+    Get the HGDP + 1KG/TGP subset release MatrixTableResource.
+
+    :param test: If true, will return the annotation resource for testing purposes
+    :param subset: The subset of the HGDP + 1KG/TGP release to return, could be "added", "subtracted", or "final"
+    :return: MatrixTableResource for specific subset
+    """
+    return VersionedTableResource(
+        default_version=CURRENT_HGDP_TGP_RELEASE,
+        versions={
+            release: TableResource(
+                f"{qc_temp_prefix(version=release) if test else f'gs://gnomad/release/{release}/ht/'}gnomad.genomes.v{release}.hgdp_1kg_subset_updated_callstats_{subset}.ht"
```

Comment:
Let's have these tables go to annotations instead of release since the call stats will be added to the release table.